DAOS-13386 cart: Execute CI tests with debug build of the UCX libraries.

jgmoore-or opened the pull request:
### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

jgmoore-or committed:
DAOS-13386 cart: upgrade to UCX 1.14.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13386 cart: Upgrade to UCX 1.14

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-scan-rpms: rpms
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13386 cart: Upgrade to UCX 1.14.

Sikip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13386 cart: Upgrade to UCX 1.14

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
Merge branch 'master' into jgm/daos-13386

jgmoore-or committed:
DAOS-13386 cart: Upgrade to UCX 1.14.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13386 cart: Test with UCX 1.14 debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:3

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,9 @@ static int data_init(int server, crt_init_options_t *opt)
 	if (server)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
```
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,9 @@ static int data_init(int server, crt_init_options_t *opt)
 	if (server)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
+        setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/1/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/1/execution/node/603/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/1/execution/node/556/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/2/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/2/execution/node/144/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/2/execution/node/601/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/2/execution/node/554/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/3/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/3/execution/node/145/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/3/execution/node/601/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/3/execution/node/585/log

jgmoore-or committed:
Merge branch 'master' into jgm/daos-13386-2

jgmoore-or committed:
DAOS-12286 cart: Execute CI tests with debug build of the UCX libraries.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:3

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/4/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/4/execution/node/145/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/5/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/5/execution/node/133/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/5/execution/node/604/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/5/execution/node/588/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/6/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/6/execution/node/173/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/6/execution/node/554/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/6/execution/node/647/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/7/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/7/execution/node/144/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/7/execution/node/554/log

jgmoore-or committed:
DAOS-13386 cart: Run CI tests with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:3

Merge branch 'master' into jgm/daos-13386-2

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/7/execution/node/647/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/8/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/8/execution/node/173/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/8/execution/node/603/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/8/execution/node/556/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/9/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/9/execution/node/556/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/9/execution/node/649/log

jgmoore-or committed:
DAOS-13386 cart: Use UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:2

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13886 cart: Test with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:3

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,10 @@ static int data_init(int server, crt_init_options_t *opt)
 	if (server)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
```
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,10 @@ static int data_init(int server, crt_init_options_t *opt)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
+        setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
```
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,10 @@ static int data_init(int server, crt_init_options_t *opt)
 	if (server)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
+        setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
+        setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
	setenv("UCX_DC_MLX5_NUM_DCI", "16", 1);
```

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/10/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/10/execution/node/145/log

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/11/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/11/execution/node/146/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/11/execution/node/604/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/11/execution/node/588/log

jgmoore-or committed:
DAOS-13386 cart: Run with UCX debug build.

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-13386 cart: Run with debug build for UCX.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:3

Merge branch 'master' into jgm/daos-13386-2

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/12/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/12/execution/node/144/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/12/execution/node/556/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/12/execution/node/603/log

jgmoore-or committed:
DAOS-13386 cart: Test with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+rc_x
PR-repos: openucx@PR-9:3

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/13/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/13/execution/node/145/log

jgmoore-or committed:
DAOS-13386 cart: Test with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+rc_x
PR-repos: openucx@PR-9:3

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented on src/cart/crt_init.c:
```diff
@@ -307,6 +307,9 @@ static int data_init(int server, crt_init_options_t *opt)
 	if (server)
 		setenv("UCX_IB_FORK_INIT", "n", 1);
 
+        setenv("D_LOG_STDERR_IN_LOG", "1", 1);
+        setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
```
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
```

daosbuild1 replied:
(style)  code indent should use tabs where possible

daosbuild1 replied:
```suggestion
	setenv("D_LOG_STDERR_IN_LOG", "1", 1);
	setenv("UCX_SOCKADDR_TLS_PRIORITY", "rdmacm", 1);
```

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/14/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/14/execution/node/146/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/14/execution/node/604/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/14/execution/node/557/log

jgmoore-or committed:
DAOS-13386 cart: Run with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+ud_x
PR-repos: openucx@PR-9:3

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12246/15/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/15/execution/node/145/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/15/execution/node/603/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/15/execution/node/587/log

jgmoore-or committed:
Merge branch 'master' into jgm/daos-13386-2

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:15

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

jgmoore-or committed:
DAOS-13386 ucx: Run with UCX debug build.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-build-leap15-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-ucx-provider: false
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: openucx@PR-9:15

Required-githooks: true

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/17/execution/node/600/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-12246/17/execution/node/601/log
